dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/SkipIfCheckpointNotSupportedAnnotationTest.java

```diff
@@ -59,7 +59,9 @@ public void testFailedCheckpointIfRunningOnNonCRIUJVM() throws Exception {
             server.setCheckpoint(new CheckpointInfo(CheckpointPhase.APPLICATIONS, false, true, false, null));
             ProgramOutput output = server.startServer(getTestMethodNameOnly(testName) + ".log");
             returnCode = output.getReturnCode();
-            assertEquals("Return code 70 expected", 70, returnCode);
+            if (!output.getStdout().contains("Unrecognized VM option")) {
```

Comment:
Instead of looking for this message, we should add `-XX:+IgnoreUnrecognizedVMOptions` to the `JVM_OPTIONS_QUOTED` variable when a checkpoint is being done:

https://github.com/OpenLiberty/open-liberty/blob/660838223dc00f7fd02fac785ab88d11878653e0/dev/com.ibm.ws.kernel.boot.ws-server/publish/bin/server#L993

This option is recognized by Hotspot/OpenJDK/OpenJ9 JVMs and will cause it to ignore the unrecognized option.

Reply:
Added, thanks!